src/Components/CreateDog/CreateDog.js

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react";
+import "./CreateDog.styles.scss";
+import axios from "axios";
+
+function DogCardInfo() {
+  const [dogName, setDogName] = useState([]);
+  const [dogInfo, setDogInfo] = useState([]);
+
+  useEffect(() => {
```

Comment:
Seperate RandomDOgInfo to new `useEffect` so it is called only once

Reply:
I separated RandomName into its own Useeffect to create 1 network request, I also put the ModifiedObject into its own useEffect to update the state variable in order to render the data into its card template correctly.  A total of 3 useEffect are being used for this component. Let me know if this is ok.

Reply:
I just wanted to move `RandomDogInfo` to a separate useEffect because it needs to be called only once. Other API calls could be together in that useEffect

Reply:
also change the name of functions to start with lower letters. Capital letter is reserved for components

Reply:
I reduced it to 2 useEffect now, since the request needs to be called every time more dogs are needed we will request more. The name Api for the moment is the only one that is requested once. 
Next, I change the capital letters as per instructed.